### PR TITLE
style(create monitor page): change form design

### PR DESCRIPTION
**Pull request by sarah-adewale**
This commit changes the design of the create monitor form and its button

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -834,9 +834,9 @@
   dependencies:
     "glob" "7.1.7"
 
-"@next/swc-win32-x64-msvc@13.1.1":
-  "integrity" "sha512-mVF0/3/5QAc5EGVnb8ll31nNvf3BWpPY4pBb84tk+BfQglWLqc5AC9q1Ht/YMWiEgs8ALNKEQ3GQnbY0bJF2Gg=="
```
Comment (cehicm) [inline yarn.lock:838]: We shouldn't commit any changes to the yarn or .env files, please revert these changes.

**Review by cehicm — CHANGES_REQUESTED** [review]
Please fix the comment and create another PR.

File: yarn.lock (hunk 1 above)
Comment (sarah-adewale) [reply yarn.lock:838]: oh wow, will do that
